docs/usage.md

```diff
@@ -110,92 +155,296 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
+
+The mean quality requirement option shared by fastp cut_front, cut_tail or cut_sliding options. Range: 1~36 (Default: 30 (Q30)).
+
+### `--qualified_quality_phred`
+
+The quality value that a base is qualified. Default 30 means phred quality >=Q30 is qualified (Default: 30).
+
+### `--unqualified_percent_limit`
+
+Percentage of bases that are allowed to be unqualified (0~100) (Default: 10).
+
+### `--min_trim_length`
+
+Reads shorter than this length after trimming will be discarded (Default: 50).
+
+### `--skip_adapter_trimming`
+
+Skip the adapter trimming step performed by fastp. Use this if your input FastQ files have already been trimmed outside of the workflow or if you're very confident that there is no adapter contamination in your data (Default: false).
+
+### `--skip_amplicon_trimming`
+
+Skip the amplicon trimming step performed by Cutadapt. Use this if your input FastQ files have already been trimmed outside of the workflow or if you're very confident that there is no primer sequence contamination in your data (Default: false).
 
-Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`.
+### `--save_trimmed`
+
+By default, trimmed FastQ files will not be saved to the results directory. Specify this flag (or set to true in your config file) to copy these files to the results directory when complete (Default: false).
+
+## Variant calling
+
+### `--callers`
+
+Specify which variant calling algorithms you would like to use. Available options are `varscan2`, `ivar` and `bcftools` (Default: 'varscan2,ivar,bcftools').
+
+### `--ivar_exclude_reads`
+
+This option unsets the `-e` parameter in `ivar trim` to discard reads without primers (Default: false).
+
+### `--filter_dups`
+
+Remove duplicate reads from alignments as identified by picard MarkDuplicates (Default: false). Note that unless you are using [UMIs](https://emea.illumina.com/science/sequencing-method-explorer/kits-and-arrays/umi.html) it is not possible to establish whether the fragments you have sequenced were derived via true biological duplication (i.e. sequencing independent template fragments) or as a result of PCR biases introduced during the library preparation.
+
+### `--min_base_qual`
+
+When performing variant calling skip bases with baseQ/BAQ smaller than this number (Default: 20).
+
+### `--min_coverage`
+
+When performing variant calling skip positions with an overall read depth smaller than this number (Default: 10).
+
+### `--max_allele_freq`
+
+Maximum allele frequency threshold for filtering variant calls (Default: 0.8).
+
+### `--save_align_intermeds`
+
+By default, intermediate [BAM](https://samtools.github.io/hts-specs/) files will not be saved. The final BAM files created after the appropriate filtering step are always saved to limit storage usage. Set to true to also save other intermediate BAM files (Default: false).
+
+### `--save_mpileup`
```

Comment:
If it's to save `pileup` files, why is the params `--save_mpileup` and not `--save_pileup`?

(I know it's a trick question, I'll probably include that in Sarek soon, and just want to be sure about the params name)

Reply:
https://www.biostars.org/p/6259/ 

> As of samtools 0.1.8 'mpileup' appears to have superseded 'pileup' for variant detection. The samtools website now has pileup as deprecated.

Reply:
Yep. The command has changed to `samtools mpileup` too.